src/orquestra/sdk/_base/_api/_wf_run.py

```diff
@@ -126,7 +128,43 @@ def by_id(
         return workflow_run
 
     @classmethod
-    def _start(cls, wf_def: ir.WorkflowDef, runtime, config, project):
+    def start_from_ir(
+        cls,
+        wf_def: ir.WorkflowDef,
+        config: t.Union[RuntimeConfig, str],
+    ):
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Does this need workspace support?